src/javascript/app/base/client.js

```diff
@@ -133,6 +133,17 @@ const Client = (() => {
         set('is_virtual', +authorize.is_virtual);
         set('session_start', parseInt(moment().valueOf() / 1000));
         set('landing_company_shortcode', authorize.landing_company_name);
+        updateAccountList(authorize.account_list);
+    };
+
+    const updateAccountList = (account_list) => {
+        account_list.forEach((account) => {
+            if (!account.excluded_until) {
+                set('excluded_until', undefined, account.loginid);
+            } else {
+                set('excluded_until', account.excluded_until, account.loginid);
```

Comment:
aren't these set on login? why do we need to set them again on authorize response?

Reply:
If the `exclude_until` is removed from back office, the user will need to log out and log in again for the Account status header to be removed. With this, it is updated after each reloads.

Reply:
I see! then, is `set('excluded_until', account.excluded_until, account.loginid);` enough instead of an if/else? if account.excluded_until is null/undefined it will just be set as null/undefined.

Reply:
and if we are doing it for excluded_until, is it worth updating all the params? the same case will happen if BO updates anything else

Reply:
@negarn That's a good idea. I have added it.